Airwallex/Card/Internal/AWXCardViewController.m

```diff
@@ -284,38 +299,39 @@ - (void)stopAnimating {
     self.confirmButton.enabled = YES;
 }
 
-- (void)setSameAsShipping:(BOOL)sameAsShipping {
-    _sameAsShipping = sameAsShipping;
-    _firstNameField.hidden = sameAsShipping;
-    _lastNameField.hidden = sameAsShipping;
-    _countryView.hidden = sameAsShipping;
-    _stateField.hidden = sameAsShipping;
-    _cityField.hidden = sameAsShipping;
-    _streetField.hidden = sameAsShipping;
-    _zipcodeField.hidden = sameAsShipping;
-    _emailField.hidden = sameAsShipping;
-    _phoneNumberField.hidden = sameAsShipping;
+- (void)setBillingInputNotRequired:(BOOL)isNotRequired {
```

Comment:
What about a more straightforward name `setBillingInputHidden:(BOOL)hidden` just for readability?